titiler/endpoints/cog.py

```diff
@@ -109,7 +101,7 @@ async def cog_metadata(
             "description": "Return an image.",
         }
     },
-    "response_class": ImgResponse,
+    "response_class": Response,
```

Comment:
There is no more need of the ImgResponse class (it was setting the cache control by default)